crates/verify/src/etherscan/mod.rs

```diff
@@ -109,14 +108,11 @@ impl VerificationProvider for EtherscanVerificationProvider {
                     if resp.result.starts_with("Unable to locate ContractCode at")
                         || resp.result.starts_with("The address is not a smart contract")
                     {
-                        warn!("{}", resp.result);
```

Comment:
actually we should log this because this usually provides useful context for debugging